tests/console/yast2_lan_device_settings.pm

```diff
@@ -90,15 +90,22 @@ sub run {
     open_yast2_lan();
 
     send_key "alt-a";    # add another device
+    assert_screen [qw(hardware-dialog device-setup-pop-up)];
```

Comment:
We execute this test on SLE 12 too, could you please test your changes for it too? (e.g. https://openqa.suse.de/tests/3264331)
We might need extra needles for it.

Reply:
Alright, makes sense, I will run a quick verification for it too. I skipped it this time because I had done extensive verification runs when I first added the test and this was a minor change.

Reply:
Ok, so the font was slightly different on SLE12-SP5: http://dreamyhamster.suse.cz/tests/786#step/yast2_lan_device_settings/49
I'm about to whip up a merge request with the additional needle and I'll update the description.

Reply:
Here we go: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/1204